camel-sap/tests/org.fusesource.camel.component.sap.test/pom.xml

```diff
@@ -177,14 +177,10 @@
         <dependency>
             <groupId>com.sap.conn.jco</groupId>
             <artifactId>sapjco3</artifactId>
-            <scope>system</scope>
```

Comment:
are you sure that we can remove the scope item? it means that we can't publish the org.fusesource.camel.component.sap.test plugin to a public location (but I guess it is already the case?)